app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_statistics.html

```diff
@@ -27,18 +27,30 @@ <h2>Statistics per case</h2>
                     {% for question in object.statistics.questions %}
                         <th>{{ question }} (GT)</th>
                     {% endfor %}
+                    {% if reader_study_display_set_view_feature %}
+                        <th>View case</th>
+                    {% endif %}
                 </tr>
             </thead>
             <tbody>
                 {% for entry in object.statistics.scores_by_case %}
                     <tr>
-                        <td data-order="{{ entry.display_set_id }}">{{ entry.display_set_id }}</td>
+                        <td data-order="{{ entry.display_set.id }}">{{ entry.display_set.id }}</td>
```

Comment:
The through relations are required for setting the workstation URL.

Reply:
I'm prefetching the display set on the view, so this does not incurr extra queries. And indeed this is necessary to set the workstation URL.

Reply:
But I see now (with the failing test) that this doesn't actually work as I thought it would.

Reply:
So then I have the option to either create a new templatetag to retrieve a display set based on its id, or to change the workstation_query to take in a pk instead of the object, but that will make retrieving a possibly defined config on the related reader study more complicated. Both solutions involve extra queries. Does anyone see a better way?

Reply:
>retrieve a display set based on its id

Why is that needed? You already have the object.

Reply:
I have pushed the version with the least additional queries. But maybe you see a better way still.

Reply:
Oh of course! Much better.